apps/events/tests/test_scheduled_messages.py

```diff
@@ -13,30 +13,6 @@
 from apps.utils.factories.experiment import ExperimentSessionFactory
 from apps.utils.time import timedelta_to_relative_delta
 
-# @pytest.fixture
-# def static_trigger(db):
-#     return StaticTriggerFactory()
-
-# def test_validation_error_raised(experiment):
-#     data = {
-#         "name": "pesky reminder",
-#         "team": experiment.team,
-#         "experiment": experiment,
-#         "trigger_event": TriggerEvent.PARTICIPANT_JOINED_EXPERIMENT,
-#         "recurring": True,
```

Comment:
We don't have a `recurring` field anymore, so the validation we did with it isn't needed anymore. We could add it to the EventAction's `param` field and create an index on that field, but something in me tell me to consider YAGNI here. My vote is to add this when/if we need it